packages/datadog-plugin-amqplib/test/integration-test/client.spec.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+const {
+  FakeAgent,
+  createSandbox,
+  checkSpansForServiceName,
+  skipUnsupportedNodeVersions,
+  spawnPluginIntegrationTestProc
+} = require('../../../../integration-tests/helpers')
+const { assert } = require('chai')
+
+const describe = skipUnsupportedNodeVersions()
+
+describe('esm', () => {
+  let agent
+  let proc
+  let sandbox
+
+  before(async function () {
+    this.timeout(20000)
+    sandbox = await createSandbox(['amqplib'], false, [`./packages/datadog-plugin-amqplib/test/integration-test/*`])
+  })
+
+  after(async () => {
+    await sandbox.remove()
+  })
+
+  beforeEach(async () => {
+    agent = await new FakeAgent().start()
+  })
+
+  afterEach(async () => {
+    proc && proc.kill()
+    await agent.stop()
+  })
+
+  context('amqplib', () => {
+    it('is instrumented', async () => {
+      const res = agent.assertMessageReceived(({ headers, payload }) => {
+        assert.propertyVal(headers, 'host', `127.0.0.1:${agent.port}`)
+        assert.isArray(payload)
+        assert.strictEqual(checkSpansForServiceName(payload, 'amqp.command'), true)
+      }, undefined)
+
+      proc = await spawnPluginIntegrationTestProc(sandbox.folder, 'server.mjs', agent.port, undefined)
```

Comment:
The final `undefined` argument here (and in most files) can probably be removed (unless the `spawnPluginIntegrationTestProc` function is using `arguments.length`?)